app/controllers/course/duplications_controller.rb

```diff
@@ -17,7 +17,10 @@ def create
 
   def authorize_duplication
     authorize!(:duplicate_from, current_course)
-    authorize!(:duplicate_to_other_instances, current_tenant) if instance_params != current_tenant.id
+    return unless instance_params != current_tenant.id
```

Comment:
Try not to use double negative. This is the same as `return if instance_params == current_tenant.id

ref: https://refactoring.com/catalog/removeDoubleNegative.html

Reply:
my bad. sorry for the oversight regarding this
though, it's quite weird that this double negative issue was not being caught by Hound earlier.
Have fixed this